docs/gridspec.rst

```diff
@@ -88,3 +88,19 @@ Example:
 
     {"grid": "H512", "ordering": "ring"}
     {"grid": "H512"}
+
+
+ORCA grid
+------------------------------------------
+
+The ``grid`` format is::
+
+    eORCAXXX_subtype
```

Comment:
ORCA grids also include a version without the "e" prefix, I think I mistakenly mentioned "W". So the full list of recognized grids is
- ORCA025_[FTUV]
- ORCA12_[FTUV]
- ORCA1_[FTUV]
- ORCA2_[FTUV]
- eORCA025_[FTUV]
- eORCA12_[FTUV]
- eORCA1_[FTUV]